fastlane/Pluginfile

```diff
@@ -6,5 +6,5 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 4.1'
 end
 
-# gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit.git', branch: 'trunk'
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 5.0'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit.git', branch: 'builkite_trigger/custom_message'
+# gem 'fastlane-plugin-wpmreleasetoolkit', '~> 5.0'
```

Comment:
🚨  Need to revert this to point to `~> 5.2` (or whatever version of the release-toolkit this ends up shipping in) before un-drafting and merging this PR